### ISSUE TITLE
Plugin should use a background thread

### ISSUE THREAD

**Issue by koraysels** [floatinghotpot/cordova-plugin-nativeaudio#85]
``` 
THREAD WARNING: ['NativeAudio'] took '134.285889' ms. Plugin should use a background thread.
```

can we expect a future release with support for seperate threads ?

**Comment by amidos2006** [comment]
Any solutions to that so far?

**Comment by koraysels** [comment]
no, and also this make s the plugin pretty useless as it jams the ui and vice versa so playback is not smooth.

**Comment by koraysels** [comment]
found this.. 
[https://cordova.apache.org/docs/en/2.8.0/guide/plugin-development/ios/#threading](https://cordova.apache.org/docs/en/2.8.0/guide/plugin-development/ios/#threading)
